ddl/delete_range.go

```diff
@@ -238,12 +239,13 @@ func (dr *delRange) doTask(ctx sessionctx.Context, r util.DelRangeTask) error {
 
 // insertJobIntoDeleteRangeTable parses the job into delete-range arguments,
 // and inserts a new record into gc_delete_range table. The primary key is
-// job ID, so we ignore key conflict error.
+// (job ID, element ID), so we ignore key conflict error.
```

Comment:
So the bug exists from day 1 of table partition?

Reply:
No, it's a long story.